addons/compat_aegis/compat_aegis_realisticnames/CfgWeapons.hpp

```diff
@@ -0,0 +1,134 @@
+class CfgWeapons {
+    class arifle_CTAR_base_F;
+    class Aegis_arifle_CTAR_tan_f : arifle_CTAR_base_F {
```

Comment:
General formatting thing, do for all of them.

```suggestion
    class Aegis_arifle_CTAR_tan_f: arifle_CTAR_base_F {
```